bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveIndicatorCommands.java

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Why have you created this interface? I mean - what is the purpose - it's not used?